spec/src/main/asciidoc/changelog.asciidoc

```diff
@@ -21,10 +21,24 @@
 
 === Release 2.0
 
+==== Incompatible changes
+
+===== Update OpenTracing API to 0.33.0 (https://github.com/eclipse/microprofile-opentracing/pull/177[#177])
+
+The following APIs were removed:
+
+* `Scope = ScopeManager.active()`: no alternative, the reference `Scope` has to be kept explicitly since the scope was created.
+* `Scope = ScopeManager.activate(Span, boolean)`: no alternative auto-finishing has been removed.
+* `Span = Scope.span()`: no alternative the reference to `Span` has to be kept explicitly since the span was started.
```

Comment:
According to the javadoc, "use Span directly or access it through ScopeManager.activeSpan()"